spec/metrics-spec.js

```diff
@@ -320,29 +362,42 @@ describe('Metrics', async () => {
   })
 
   describe('reporting pane items', async () => {
-    describe('when the user is NOT chosen to send events', async () => {
+    describe('when shouldIncludePanesAndCommands is false', async () => {
       beforeEach(async () => {
         spyOn(Reporter, 'sendPaneItem')
+        spyOn(Reporter, 'sendEvent')
 
         const {mainModule} = await atom.packages.activatePackage('metrics')
+
         mainModule.shouldIncludePanesAndCommands = false
 
         await conditionPromise(() => Reporter.request.callCount > 0)
       })
 
       it('will not report pane items', async () => {
+        Reporter.addCustomEvent.reset()
```

Comment:
Given that we're [calling this in the `afterEach` function](https://github.com/atom/metrics/blob/a9afa070ff8b1d82815012f9a1da0c4c369a86b0/spec/metrics-spec.js#L34) for all tests in this file, can we remove this line, or do we need it?

Reply:
ah, these were some debug code trying to get to the bottom of my comment re file open events.  Will remove them.